packages/react/src/hooks/useAuth.ts

```diff
@@ -102,18 +126,39 @@ export const useAuth: UseAuth = () => {
       sessionId,
       userId,
       actor: null,
+      orgId: null,
+      orgRole: null,
+      orgSlug: null,
+      signOut,
+      getToken,
+    };
+  }
+
+  if (!!sessionId && !!userId && !!orgId && !!orgRole && orgSlug !== undefined) {
```

Comment:
What happens if I have an active org but orgSlug is undefined? I think an org can exist without a slug, so will we be hitting the `  throw new Error(invalidStateError);` error?

Reply:
Should be null in that case. Pretty sure I checked this, but I can check again. We could just do `orgSlug || null` and not have it in the if statement as well though.

Reply:
I think this should be the way to go. Otherwise all orgs without slug will throw an error right now. The `|| null` fallback sounds reasonable

Reply:
Done. Please check!

Reply:
I can confirm that it would be ok either way since the slug is null in the response from the backend! I've also added null as a possible value for the slug in the `OrganizationResource` type since it was missing.